v7.h

```diff
@@ -23,7 +23,10 @@
 extern "C" {
 #endif /* __cplusplus */
 
+#include <stddef.h>   /* For size_t */
+
 #define V7_VERSION "1.0"
+#define V7_GENERIC_OBJECT_PROTOTYPE 0     /* For v7_create_object() */
```

Comment:
Cause V7_GENERIC_OBJECT_PROTOTYPE is defined in public v7.h
In order to expose V7_NULL, we need to expose quite a bit of stuff to the user - our packing mechanism.